src/commands/release-notes/GenerateReleaseNotes.ts

```diff
@@ -146,21 +180,56 @@ export class GenerateReleaseNotes implements ICommand {
         return result;
     }
 
-    private async generateChangelog(file: ReleaseNotesMessagesFile, explicits: ReleaseNotesMessagesFile | null, log: IGitLogEntry[]): Promise<void> {
-        const changelog = [`# Changelog ${new Date().toDateString()}`, ''];
-
-        changelog.push('');
-        changelog.push(`_Commit range: ${this.fromHash} - ${this.toHash}_`);
-        changelog.push('');
-
+    private generateChangelog(file: ReleaseNotesMessagesFile, explicits: ReleaseNotesMessagesFile | null, log: IGitLogEntry[]): void {
+        this.changelog = [`_Changelog created on ${new Date().toDateString()}_`];
```

Comment:
Please check format for backwards-compatibility with the html we publish to central.
[release notes example in central](https://central.collaboration-factory.de/pages/yglclatbg0lps2xgqbfurr778/22.4?highlightedEntityUid=page%2Fyglclatbg0lps2xgqbfurr778),
See also [the script in the cplace orb](https://github.com/collaborationFactory/cplace-circleci-orb/blob/master/src/scripts/publish_release_notes.sh).